var/spack/repos/builtin/packages/py-amrex/package.py

```diff
@@ -60,10 +65,10 @@ class PyAmrex(CMakePackage, PythonExtension, CudaPackage, ROCmPackage):
 
     depends_on("cxx", type="build")
 
-    depends_on("cmake@3.20:", type="build")
+    depends_on("cmake@3.24:", type="build")
```

Comment:
I didn't check the older version but the newer one puts a limit on `cmake` per `pyproject.toml`, along the lines:

```suggestion
    depends_on("cmake@3.20:", type="build", when="@:24.08")
    depends_on("cmake@3.24:3", type="build", when="@24.10")
```

Reply:
Though `setup.py` does not enforce it.

Reply:
Oh yes, I just bumped it up to `3.24+` because that fits as well, but I can explicitly write
```suggestion
    depends_on("cmake@3.20:3", type="build", when="@:24.08")
    depends_on("cmake@3.24:3", type="build", when="@24.09:")
```
as well.